### ISSUE TITLE
Login Broken

### ISSUE THREAD

**Issue by jamiejackson** [atorres1929/PandoraInfoGetter#1]
Maybe Pandora has changed things up. This doesn't seem to work.

```
Pandora login failed, check email and password
Login credentials wrong! Try again!
```

**Comment by atorres1929** [comment]
Checking it out now.

**Comment by atorres1929** [comment]
@jamiejackson looks like it is broken, they changed things up. I'm gonna have to take another look at this. I'll probably end up converting this project to a website.

In the mean time, take a look at this: http://pandorasongs.oliverzheng.com/

There are a lot of tools, to convert from pandora to spotify.

**Comment by jamiejackson** [comment]
For what it's worth, I ended up using an online converter, Pandify. Something to note is that it didn't require a Pandora password, since it relied on the public profile. For me, that wasn't a problem

**Comment by atorres1929** [comment]
Yeah I just saw that, think I'll just scrap the project actually since there are other better solutions. No need to reinvent the wheel.
